src/money.jsx

```diff
@@ -121,7 +122,11 @@ export default class ApiMakerInputsMoney extends React.Component {
   }
 
   inputId() {
-    return `${this.props.model.modelClassData().paramKey}_${inflection.underscore(this.props.attribute)}`
+    if (this.props.id) {
```

Comment:
This will make it possible to pass undefined and disable the ID attribute :-) So even if it is false'able it will still be passed on.
```suggestion
    if ("id" in this.props) {
```

Reply:
> This will make it possible to pass undefined and disable the ID attribute :-) So even if it is false'able it will still be passed on.

So just like now. But I guess we could make it entirely optional doing as you suggest. 

Realistically, we should probably use `ApiMakerIdForComponent` so it follows the same rules as `ApiMakerInput`. How does that sound?

Reply:
I totally agree :-)